sdks/python/apache_beam/io/gcsio.py

```diff
@@ -47,7 +47,7 @@
       'Google Cloud Storage I/O not supported for this execution environment '
       '(could not import storage API client).')
 
-DEFAULT_READ_BUFFER_SIZE = 1024 * 1024
+DEFAULT_READ_BUFFER_SIZE = 16 * 1024 * 1024
```

Comment:
Do we know how much it would improve if we increase this to an higher number instead of 16? I assume 16 is a good trade off between memory usage and read speed, but have we explored other options?